cluster-sync/sync.sh

```diff
@@ -168,7 +168,7 @@ if [ "${KUBEVIRT_PROVIDER}" != "external" ]; then
   _kubectl apply -f "./_out/manifests/file-host.yaml"
   _kubectl apply -f "./_out/manifests/registry-host.yaml"
   # Imageio test service:
-  _kubectl apply -f "./_out/manifests/imageio.yaml"
+#  _kubectl apply -f "./_out/manifests/imageio.yaml"
```

Comment:
Why is this commented, it is needed for the imageio tests.

Reply:
Some experiments in totally unrelated change. 
Restored.